client/me/purchases/remove-purchase/index.jsx

```diff
@@ -176,28 +355,273 @@ const RemovePurchase = React.createClass( {
 							comment: "'it' refers to a product purchased by a user"
 						} )
 					}
+
 				</p>
 
 				{ includedDomainText }
 			</div>
 		);
 	},
 
+	renderQuestionOne() {
+		const reasons = {},
+			orderedReasons = [];
+
+		const couldNotInstallInput = (
+			<FormTextInput
+				className="remove-purchase__reason-input"
+				name="couldNotInstallInput"
+				id="couldNotInstallInput"
+				value={ this.state.questionOneText }
+				onChange={ this.handleTextOne }
+				placeholder={ this.translate( 'What plugin/theme were you trying to install?' ) } />
+		);
+		reasons.couldNotInstall = (
+			<FormLabel key="couldNotInstall">
+				<FormRadio
+					name="couldNotInstall"
+					value="couldNotInstall"
+					checked={ 'couldNotInstall' === this.state.questionOneRadio }
+					onChange={ this.handleRadioOne } />
+				<span>{ this.translate( 'I couldn\'t install a plugin/theme I wanted.' ) }</span>
+				{ 'couldNotInstall' === this.state.questionOneRadio ? couldNotInstallInput : null }
```

Comment:
There's a neat shorthand for conditionally including content that's a little nicer than a ternary.
```js
{ booleanCheck && Content }
```
so here it would be:
```js
{ 'couldNotInstall' === this.state.questionOneRadio && couldNotInstallInput }
```